### DISCUSSION TITLE
Compliant Secure Computer For Users Not Engineers

### DISCUSSION THREAD

**Discussion by abakasam** [celzero/rethink-app#345] [Ideas]
Why not engineers? You need mail for engineers. You need attack for hackers. You need sup for scientist.

Difficult | Defend      | Mail          | Unpack        | Attack        | Sup
----------|------------ | ------------- | ------------- | ------------- | -------------
Easy      | Lock        |               | Move          |
Normal    | Block       |               | Scan          |
Hard      | Map         |               | Contact       |

* Defend
  * DNS [rethinkdns], SOCKS5 Proxy [rethinkdns]  
  * Rules (Universal [rethinkdns], All Apps [rethinkdns])
  * Network Log [rethinkdns] #283 

* Unpack
  * HTTPS [internet], HTTP Proxy [rethinkdns]
  * Scan [nortan]
  * Contact [coresecurity] https://www.coresecurity.com/services

**Comment by ignoramous** [comment]
Hi: Sorry, I couldn't quite catch the intent of this issue?

**Comment by abakasam** [comment]
Recent issue of HTTP Proxy, and Network Map... This issue is crying for a repeated scheduled request to a Nortan application for a scan of incoming packets not the system, and then a sample request to be sent to Core Security by a in application form.

Sam

**Comment by ignoramous** [comment]
@abakasam Is what you're trying to say here is that engs, scientists, hackers, and regular users need different (easy, medium, hard) functionality from such an app, and that those functionalities are missing?

Well, even though this software is an on-going, in-progress effort, to tick all those boxes for various user-personas is going to take an insurmountable amount of person-years that this open source project simply doesn't have.

**Comment by abakasam** [comment]
Two feature request. #283

A Use of RethinkDNS Network Log to create Threatmap
B Virus scan then posting of log on one site

![IMG_20220107_114009__01](https://user-images.githubusercontent.com/58202540/148584761-de0bd482-af92-4f14-a356-8ab35649ea7f.jpg)
